addons/mail/static/tests/tours/discuss_mention_suggestions_group_restricted_channel.js

```diff
@@ -0,0 +1,17 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("discuss_mention_suggestions_group_restricted_channel.js", {
```

Comment:
@seb-odoo @Atovange  example of useless test that should be replaced by small unit tests. We are now going to take 2 seconds / each build (aka probably hours of runbot each day) for something that could take a few ms by py. This kind of niche issue does not require an extensive tour.